docker-compose.yml

```
@@ -0,0 +1,13 @@
+# Base configuration
+# Keys without a value are passed into the config via a .env file in the same
+# directory as the `docker-compose` command was run from
+version: "2"
+services:
+  app:
+    image: coach
```

We might as well update this to the name of the image that will be created after you merge:

```suggestion
    image: ghcr.io/ohsucmp/coach
```